src/components/Dashboard/TodoList.vue

```diff
@@ -680,11 +680,8 @@ export default class TodoList extends Vue {
       }
     }
 
-    // Add todo items count +1 when SP/GP with compliance warning
-    const todoLength = this.isActionRequired ? this.todoItems.length + 1 : this.todoItems.length
```

Comment:
This was because the previous panel counted as a Todo item. Now that panel (alert) is in its own section above the Todo section.